Jkadhuwa/quick-credit
# #166103736 Admin can verify users

**What does this PR do?**

- Admin can verify users

**Description of Task to be completed?**

- Create mocha test for the endpoint

- Create an endpoint for admin to verify users

**How should this be manually tested?**

- Using postman run hhtp://localhost:8080/api/v1/users/userEmail/verify

- Then send the request using PATCH method  by pressing the send button

**What are the relevant pivotal tracker stories?**

- #166103736

**Any background context you want to add?**

- NIL

**Screenshots**

- NIL
